pkg/model/event.go

```diff
@@ -18,11 +18,16 @@ import (
 	"crypto/sha256"
 	"encoding/hex"
 	"fmt"
+	"regexp"
 	"sort"
 	"strings"
 )
 
-var hashFunc = sha256.New
+var (
+	hashFunc = sha256.New
+
+	eventKeyFormatRegex = regexp.MustCompile(`^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$`)
```

Comment:
[nits] This variable seems unused

Reply:
Fixed 🙏 [341ce20](https://github.com/pipe-cd/pipecd/pull/5339/commits/341ce208e39165a07744faf376c687250861be95)